src/main/java/org/example/Hobby.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.util.ArrayList;
+
+public class Hobby {
+    private String nameOfHobby;
+    private int frequencyPerWeek;
+    ArrayList<Address> addressesForHobby = new ArrayList<>();
+
+    public String getNameOfHobby() {
+        return nameOfHobby;
+    }
+
+    public void setNameOfHobby(String nameOfHobby) {
+        this.nameOfHobby = nameOfHobby;
+    }
+
+    public int getFrequency() {
+        return frequencyPerWeek;
+    }
+
+    public void setFrequency(int frequency) {
+        this.frequencyPerWeek = frequency;
+    }
+    /**
+     * Returns a Hobby object which has a name and a frequency
+     * @param  nameOfHobby  the name of a Hobby we want to create
```

Comment:
please don't javadoc short methods, document the more complicated ones, if the don't exist, write a summary at the beginning of the class